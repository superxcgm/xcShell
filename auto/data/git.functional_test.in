mkdir git_test
cd git_test
git init | grep -v 'hint:'
git checkout -b not_master
mkdir subdir
cd subdir
cd ../.git