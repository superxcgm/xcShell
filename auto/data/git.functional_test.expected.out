xcShell_functional_test > mkdir git_test
xcShell_functional_test > cd git_test
git_test > git init > /dev/null 2>&1
Initialized empty Git repository in /tmp/xcShell_functional_test/git_test/.git/
git_test[31m git(master)[0m > git checkout -b not_master
Switched to a new branch 'not_master'
git_test[31m git(not_master)[0m > mkdir subdir
git_test[31m git(not_master)[0m > cd subdir
subdir[31m git(not_master)[0m > cd ../.git
.git[31m git(not_master)[0m > 